_maps/map_files/BoxStation/BoxStation.dmm

```diff
@@ -32502,21 +32257,18 @@
 /obj/structure/cable{
 	icon_state = "1-8"
 	},
-/obj/structure/disposalpipe/segment,
 /obj/structure/cable{
 	icon_state = "1-2"
 	},
 /turf/open/floor/plasteel/white,
 /area/science/research)
 "bBF" = (
-/obj/machinery/portable_atmospherics/scrubber,
-/obj/item/storage/firstaid/toxin,
-/obj/effect/turf_decal/stripes/line,
-/obj/machinery/airalarm/unlocked{
```

Comment:
Please dont replace this with an locked airalarm

Reply:
it's now unlocked in meta